ingress/controller.go

```diff
@@ -70,6 +70,14 @@ func (c *controller) Start() error {
 
 	go c.watchForUpdates()
 
+	frontends, err := c.frontend.Attach()
+
+	if err != nil {
+		return fmt.Errorf("unable to attach to front end %v", err)
+	}
+
+	log.Infof("Attached to %d frontend(s)", frontends)
```

Comment:
Would it be better to just log this in the frontend itself? That way we're not returning a value just so it can be logged, Attach can just return the error.